examples/sklearn-kmeans/job_configs/sklearn_kmeans_base/app/custom/kmeans_assembler.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# We will move to this app_common when it gets matured
+from typing import Dict
+
+import numpy as np
+from app_common.aggregators.assembler import Assembler
+from nvflare.apis.dxo import DataKind
+from sklearn.cluster import KMeans
+
+
+class KMeansAssembler(Assembler):
+    def __init__(self):
+        super().__init__(data_kind=DataKind.WEIGHTS)
+        # Aggregator needs to keep record of historical
+        # center and count information for mini-batch kmeans
+        self.center = None
+        self.count = None
+        self.n_cluster = 0
```

Comment:
do we need this "n_cluster" as instance variable?

Reply:
I think so, needed for both train and evaluation